packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -482,17 +519,20 @@ class _DraggableState<T extends Object> extends State<Draggable<T>> {
   }
 
   _DragAvatar<T>? _startDrag(Offset position) {
+    final Offset dragStartPoint;
```

Comment:
nit: Why move this up?

Reply:
i don't know. maybe my urge to have declarations on top. moved it back.